build/paths.cake

```diff
@@ -40,6 +40,7 @@ public class BuildPaths
         var releaseNotesOutputFilePath = buildArtifactDir.CombineWithFilePath("releasenotes.md");
 
         var vsixOutputFilePath = buildArtifactDir.CombineWithFilePath("gittools.gitversion-" + semVersion + ".vsix");
+        var vsixNetCoreOutputFilePath = buildArtifactDir.CombineWithFilePath("gittools.gitversion_netcore-" + semVersion + ".vsix");
```

Comment:
The path here should match the artifact's path for publishing

Reply:
basically just change `gitversion_netcore` with `gitversion-netcore`

Reply:
nice find!  Done and pushed.